tests/codegen_testcases/solidity/selector_adds_non_base_function.sol

```diff
@@ -0,0 +1,19 @@
+// RUN: --target polkadot --emit cfg
+
+// This test makes sure that we're not including the function a in contract C
+
+// NOT-CHECK: C::A::function::a
```

Comment:
This test does not fail if I comment the changes in `src/codegen/external_functions.rs`

Reply:
Good point, fixed now